model/accounts_mgmt/v1/summary_metrics_type.model

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+struct SummaryMetrics {
+	Name string
+	Vector []SummaryVector
```

Comment:
I know the dashboard is already functioning, so this might be too late, but still:

A "vector" being an array of "vector" is confusing :confused:.
- Should this field been called `Vectors`?
- Alternatively, "vector" is OK to describe an array, if elements had some "scalar" name.  Each array element contains {Time, Value} so maybe they should be called something like `SummarySample`?

Reply:
Let's see:
```
$ ocm get /api/accounts_mgmt/v1/organizations/(ocm whoami | jq .organization.id --raw-output)/summary_dashboard
{
  "id": "summary",
  "kind": "Dashboard",
  "href": "/api/accounts_mgmt/v1/organizations/1H1PQMDtwzAUsjPxgoWRjhSpNGD/summary_dashboard",
  "name": "summary",
  "metrics": [
    {
      "name": "clusters_total",
      "vector": [
        {
          "time": "2020-09-10T16:06:38.152249168Z",
          "value": 1950
        }
      ]
    },
    {
      "name": "sum_used_cpu",
      "vector": [
        {
          "time": "2020-09-10T16:06:38.152249168Z",
          "value": 1626.4422826738557
        }
      ]
    },
    {
      "name": "sum_total_cpu",
      "vector": [
        {
          "time": "2020-09-10T16:06:38.152249168Z",
          "value": 14285
        }
      ]
    },
    {
      "name": "sum_used_memory",
      "vector": [
        {
          "time": "2020-09-10T16:06:38.152249168Z",
          "value": 8638143479808
        }
      ]
    },
    {
      "name": "sum_total_memory",
      "vector": [
        {
          "time": "2020-09-10T16:06:38.152249168Z",
          "value": 60323344625664
        }
      ]
    },
    {
      "name": "unhealthy_clusters_total",
      "vector": [
        {
          "time": "2020-09-10T16:06:38.152249168Z",
          "value": 122
        }
      ]
    },
    {
      "name": "connected_clusters_total",
      "vector": [
        {
          "time": "2020-09-10T16:06:38.152249168Z",
          "value": 245
        }
      ]
    },
    {
      "name": "clusters_up_to_date_total",
      "vector": [
        {
          "time": "2020-09-10T16:06:38.152249168Z",
          "value": 62
        }
      ]
    },
    {
      "name": "clusters_upgrade_available_total",
      "vector": [
        {
          "time": "2020-09-10T16:06:38.152249168Z",
          "value": 183
        }
      ]
    }
  ]
}
```
OK, each `vector` is a time series, which leaves future ability to include multiple samples over time, although all contain 1 sample presently.
=> I think `Vector` field name is fine but `SummaryVector` type should be renamed.